### DISCUSSION TITLE
CAPTCHA on login screen

### DISCUSSION THREAD

**Discussion by Manishkparmar80** [Budibase/budibase#1695] [Ideas]
CAPTCHA on login screen is very much essential specially for app which are public facing. 

May Consider including this functionality in future release.

**Comment by mjashanks** [comment]
Hey @Manishkparmar80, thanks for the suggestion.

Can you help me understand the need for this a bit more? Our thoughts around this are that all logins to budibase would be controlled by the admins of the system - added one at a time, or via an internal authentication integration.

So, by definition, all logins have a certain level of trust attached to them. For example, external user "sign-up" is not something that's on our radar.

For me, CAPTCHA is used to protect against bots - I can't see bots being an issue with a system like Budibase. Happy to be proved wrong, if you could give some more info :)

**Comment by slfhstr** [comment]
Captchas do nothing to improve real security, just reduce bots.
And they're effing annoying.
Better to consider/require 2FA / MFA
